Content.Server/GameObjects/Components/Projectiles/StunProjectileComponent.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using Content.Server.GameObjects.Components.Mobs;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Log;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.GameObjects.Components.Projectiles
+{
+    /// <summary>
+    /// Adds stun when it collides with an entity
+    /// </summary>
+    [RegisterComponent]
+    public sealed class StunProjectileComponent : Component, ICollideBehavior
```

Comment:
Since "Stun" is actually something you can do with the `StunnableComponent`, and this does *not* inflict stun, I think this name for it is confusing. We don't want to inherit the SS13 problem where the names for the different stun/paralyze/knockdown types are mixed up. If this paralyzes, it should be called `ParalyzeProjectileComponent`.

Reply:
Would it be better to add the other stunnable types to it and just call it "StunnableProjectile"?

Reply:
That would work for me.